common/src/main/java/net/opentsdb/utils/Comparators.java

```diff
@@ -60,4 +62,27 @@ public int compare(final Map<K, V> a, final Map<K, V> b) {
     }
     
   }
+
+  public static class ListComparison {
+
+    public static <T> boolean equalLists(List<T> one, List<T> two){
```

Comment:
See if you can make this compare Objects and still work without generics so that we can avoid instantiating comparison instances.

Reply:
And javadoc :)